lte/cloud/go/services/lte/obsidian/handlers/handlers_test.go

```diff
@@ -4425,3 +4443,28 @@ func newMutableGateway(id string) *lteModels.MutableLteGateway {
 	}
 	return gw
 }
+
+func newGatewayResponse(id string) *lteModels.LteGateway {
+	checkedInRecently := lteModels.GatewayCheckedInRecently(false)
+	gw := &lteModels.LteGateway{
+		Device: &models.GatewayDevice{
+			HardwareID: id + "_hwid",
+			Key:        &models.ChallengeKey{KeyType: "ECHO"},
+		},
+		ID:          models2.GatewayID(id),
+		Name:        "foobar",
+		Description: "foo bar",
+		Magmad: &models.MagmadGatewayConfigs{
+			CheckinInterval:         15,
+			CheckinTimeout:          10,
+			AutoupgradePollInterval: 300,
+			AutoupgradeEnabled:      swag.Bool(true),
+		},
+		Cellular:               newDefaultGatewayConfig(),
+		ConnectedEnodebSerials: []string{},
+		Tier:                   "t0",
+		ApnResources:           lteModels.ApnResources{},
+		CheckedInRecently:      &checkedInRecently,
```

Comment:
why not just 

```
CheckedInRecently: &lteModels.GatewayCheckedInRecently(false)
```

Since this is a test it is always good to see the object at one sight.

Reply:
Changed.